docs/data/material/components/icons/icons.md

```diff
@@ -26,23 +26,17 @@ You can [search the full list of these icons](/material-ui/material-icons/).
 
 Run one of the following commands to install it and save it to your `package.json` dependencies:
 
-:::info
-The `next` tag is used to download the latest <b>pre-release</b>, v6 version. Remove it to get the current stable version.
-:::
-
-<!-- #default-branch-switch -->
```

Comment:
Lets keep this one (same as https://github.com/mui/material-ui/pull/43454/files#r1731548827)